src/css/overrides.scss

```diff
@@ -1,5 +1,41 @@
+%media {
```

Comment:
I got two issues with that 😢 :

## The naming

The placeholder name `%media` is very vague 😞 (it's unclear which properties it affects or what it does).
A more descriptive name could fix that (e.g. `%max-width-breakpoints` or `%media-max-width`)

## The overall idea
I've looked into tailwind docs and I've found all the necessary customization options:
1. That one allows breakpoints customization https://tailwindcss.com/docs/breakpoints
2. That one allows container size & padding customization https://tailwindcss.com/docs/container

TL;DR I suggest customizing containers behavior with `tailwind.config.js`